modules/core/test/v2/unit/wallet.ts

```diff
@@ -590,6 +590,42 @@ describe('V2 Wallet:', function() {
     }));
   });
 
+  describe('maxNumInputsToUse verification', function() {
+    const address = '5b34252f1bf349930e34020a';
+    const maxNumInputsToUse = 2;
+    let basecoin;
+    let wallet;
+
+    before(co(function* () {
```

Comment:
we can use `async function ()` now can't we?

Reply:
yes, we can. I would accept either coroutines or async/await.

Reply:
I've added a second commit that replaces promises in this test with async-await.